tests/functional/test_study_version_api.py

```diff
@@ -0,0 +1,115 @@
+# pylint: disable=too-many-lines
+"""Tests for the Study Metadata API endpoints"""
+import json
+
+import pytest
+
+# ------------------- VERSION ADD ------------------- #
+
+
+def test_get_version_study_metadata(_logged_in_client):
+    """
+    Given a Flask application configured for testing
+    WHEN the /study/{study_id}/dataset/{dataset_id}/version/{version_id}/study-metadata endpoint is requested (GET)
+    THEN check that the response is valid and retrieves the design metadata
+    """
+    study_id = pytest.global_study_id["id"]  # type: ignore
+    dataset_id = pytest.global_dataset_id  # type: ignore
+    version_id = pytest.global_version_id  # type: ignore
+    response = _logged_in_client.get(
+        f"/study/{study_id}/dataset/{dataset_id}/version/{version_id}/study-metadata"
+    )
+
+    assert response.status_code == 200
+
```

Comment:
We might need to test the contents as well here. Especially since the front end is expecting a certain structure